server/db/migrations/19_add_indexes_to_container_log.rb

```diff
@@ -1,8 +1,6 @@
 class AddIndexesToContainerLog < Mongodb::Migration
   def self.up
-    Thread.new {
-      Thread.current.abort_on_exception = true
-      ContainerLog.create_indexes # might take a long time
-    }
+    info "recreating container log indexes, this might take long time"
+    ContainerLog.create_indexes
```

Comment:
If we're blocking the migrations for the duration of the index create, then should we also revert the `index(..., { background: true })` mode as well? It sounds like the normal foreground index builds are better in that case: https://docs.mongodb.com/v3.0/tutorial/build-indexes-in-the-background/

> Background index builds take longer to complete and result in an index that is initially larger, or less compact, than an index built in the foreground.